spec/facades/quote_facade_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'QuoteFacade' do
```

Comment:
Nice work testing what you need to for the Facade and nothing more. I also like the use of `be_all`. I forget about that one and this is such an elegant way of testing that.